charts/datadog/templates/_container-security-agent.yaml

```
@@ -37,6 +37,14 @@
     {{- if .Values.datadog.securityAgent.compliance.xccdf.enabled }}
     - name: DD_COMPLIANCE_CONFIG_XCCDF_ENABLED
       value:  {{ .Values.datadog.securityAgent.compliance.xccdf.enabled | quote }}
+    - name: DD_COMPLIANCE_CONFIG_HOST_BENCHMARKS_ENABLED
+      value:  {{ .Values.datadog.securityAgent.compliance.xccdf.enabled | quote }}
+    {{- end }}
+    {{- if .Values.datadog.securityAgent.compliance.host_benchmarks.enabled }}
+    - name: DD_COMPLIANCE_CONFIG_XCCDF_ENABLED
+      value:  {{ .Values.datadog.securityAgent.compliance.host_benchmarks.enabled | quote }}
+    - name: DD_COMPLIANCE_CONFIG_HOST_BENCHMARKS_ENABLED
+      value:  {{ .Values.datadog.securityAgent.compliance.host_benchmarks.enabled | quote }}
     {{- end }}
```

We may be able to factorize those two blocks:

```suggestion
    {{- if .Values.datadog.securityAgent.compliance.xccdf.enabled || .Values.datadog.securityAgent.compliance.host_benchmarks.enabled }}
    - name: DD_COMPLIANCE_CONFIG_XCCDF_ENABLED
      value:  "true"
    - name: DD_COMPLIANCE_CONFIG_HOST_BENCHMARKS_ENABLED
      value:  "true"
    {{- end }}
```

---

Done. Thanks